nanohttp/tests/test_exceptions.py

```diff
@@ -109,3 +111,23 @@ def index(self):
     with pytest.raises(E):
         Given(Root(), configuration='debug: false')
 
+
+@pytest.mark.parametrize('http_success_exception', (
+    HTTPNoContent,
+    HTTPCreated,
+    HTTPAccepted,
+    HTTPNonAuthoritativeInformation,
+    HTTPResetContent,
+    HTTPPartialContent
```

Comment:
Please add a `,` to end of the line.

Reply:
@mohammadsheikhian  This change is redundant, can you refer the guideline?

Reply:
The `,` will make future changes easier. like remove, add or replace a line. The diff log will show one line instead of two lines.